packages/image_picker/image_picker/example/ios/RunnerTests/ImageUtilTests.m

```diff
@@ -6,6 +6,7 @@
 
 @import image_picker;
 @import XCTest;
+#import <image_picker/FLTImagePickerImageUtil.h>
```

Comment:
Isn't this still relying on the bug in the framework (that it includes this header in the packaged framework at all)?

Pre module-map I would expect this to just be `#import "FLTImagePickerImageUtil.h"` with appropriate include paths for the target. I expect there's probably a cleaner way using the Test module though?

Reply:
@mvanbeusekom i think you can put this header under the submodule `Test` that you've created, and just do `@import image_picker.Test;` here.

Reply:
The solution proposed by @hellohuanlin worked like a charm. Thanks